airbyte-workers/src/main/java/io/airbyte/workers/process/IntegrationLauncher.java

```diff
@@ -7,6 +7,11 @@
 import io.airbyte.workers.WorkerException;
 import java.nio.file.Path;
 
+/**
+ * This interface provides an abstraction for launching a container that implements the Airbyte
+ * Protocol. Such containers implement each method that is defined in the Protocol. This class,
+ * provides java methods to invoke the methods on these containers.
+ */
 public interface IntegrationLauncher {
```

Comment:
What exactly is the `jobRoot` path that these methods take in?

Reply:
good question. added this: 

> Each method takes in a jobRoot that is a directory where the worker that runs the method can use as temporary file system storage.